flow360/component/simulation/meshing_param/params.py

```diff
@@ -70,9 +72,32 @@ class MeshingParams(Flow360BaseModel):
         + "However the impact on regions without close proximity is negligible.",
     )
 
-    surface_layer_growth_rate: float = pd.Field(
+    ##------------ Default settings ------------##
+
+    ##::   Default surface edge settings
+    default_surface_edge_growth_rate: float = pd.Field(
         1.2, ge=1, description="Global growth rate of the anisotropic layers grown from the edges."
-    )  # Conditionally optional
+    )
+
+    ##::    Default boundary layer settings
+    default_boundary_layer_growth_rate: float = pd.Field(
+        1.2, description="Default growth rate for volume prism layers.", ge=1
+    )
+    default_boundary_layer_first_layer_thickness: Optional[LengthType.Positive] = pd.Field(
+        description="Default first layer thickness for volumetric anisotropic layers.", ge=1
+    )  # Truly optional if all BL faces already have first_layer_thickness
+
+    ##::    Default surface layer settings
+    default_surface_max_edge_length: Optional[LengthType.Positive] = pd.Field(
```

Comment:
We cannot figure out if `default_surface_max_edge_length` is actually needed until we know if we need to run surface mesh. This correlates with this ticket: https://flow360.atlassian.net/browse/SCFD-2519

Reply:
Ideally we should make sure that user can click the run button if everything will work. Not just passing validation.

Reply:
proposition:
```python
class MeshingParams:
  defaults: MeshingDefaults
  volume_zones
  refinements
  
class MeshingDefaults:
    boundary_layer_growth_rate
    surface_mesh_growth_rate
    curvature_resolution_angle
    surface_max_edge_length
    boundary_layer_first_layer_thickness
```

Reply:
Changed